src/angular/src/app/spark-core-angular/components/sprk-dictionary/sprk-dictionary.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'sprk-dictionary',
+  template: `
+    <div [ngClass]="getClasses()">
+    <dl class="sprk-c-Dictionary__keyvaluepairs">
+      <div *ngFor="let key of objectKeys(data)" class="sprk-c-Dictionary__keyvaluepair">
+        <dt class="sprk-c-Dictionary__key sprk-b-Label">{{ key }}</dt>
+        <dd class="sprk-c-Dictionary__value">{{ data[key] }}</dd>
+      </div>
+    </dl>
+  </div>`
+})
+
+export class SparkDictionaryComponent {
+  @Input() data: Object = {};
+  @Input() dictionaryType: string = 'base';
+  @Input() additionalClasses: string;
+  objectKeys: Function = Object.keys;
+
+  getClasses(): string {
+    let classArray: Array<String> = [
+      'sprk-c-Dictionary'
+    ];
+
+    if (this.dictionaryType === 'striped') {
```

Comment:
This seems good because if they put the value of dictionaryType as "basic", which is the default, it will still work.